src/libs/error.ts

```diff
@@ -3,7 +3,7 @@ import _ from 'lodash/fp';
 import { sessionTimedOutErrorMessage } from 'src/auth/auth-errors';
 import { notify } from 'src/libs/notifications';
 
-export const reportError = async (title, obj) => {
+export const reportError = async (title, obj?: unknown) => {
```

Comment:
Do we ever omit the `obj` argument?

Reply:
lol, yeah, in this PR in fact... useWorkspaces hook does, with that line of code being based on the original hook's code

`      void reportError('Error loading workspace list');`

Reply:
That's kind of a break with convention. Usually we include the error via `withErrorReporting`.

Reply:
meh... just kept it as-is... error reporting/handling is it's own project to sink our teeth into...

Reply:
For what it's worth, It's more than a break with code style convention, it's a break with existing UX patterns. Not passing the error prevents the user from being able to see information about the error.

Reply:
yes, I TOTALLY agree.... but that's a sweater I don't want to pull on yet...